arches/app/src/arches/themes/default.ts

```diff
@@ -0,0 +1,81 @@
+import { definePreset, palette } from "@primevue/themes";
+import Aura from "@primevue/themes/aura";
+
+const archesColors = Object.freeze({
+    blue: "#579ddb",
+    green: "#3acaa1",
+    red: "#f75d3f",
+});
+
+export const ArchesPreset = definePreset(Aura, {
+    primitive: {
+        arches: {
+            ...archesColors,
+            legacy: {
+                sidebar: "#2d3c4b",
+            },
+        },
+        green: palette(archesColors.green),
+        red: palette(archesColors.red),
+    },
+    semantic: {
+        primary: palette(archesColors.blue),
+        navigation: {
+            list: {
+                padding: "0",
+            },
+            item: {
+                padding: "1rem",
+            },
+            color: "{arches.legacy.sidebar}",
+        },
+        iconSize: "small",
```

Comment:
I would recommend remove this, I don't believe we should have our "default" icon size be small ( or that we should offer a "default" icon size)

Reply:
PrimeVue themes come with an iconSize whether we like it or not (1rem), so I have a feeling we will want to override it with _something_. But we can definitely punt.